client/my-sites/plans-v2/style.scss

```diff
@@ -15,6 +15,19 @@
 		& > .details__column:not( :first-of-type ) {
 			margin-left: 24px;
 		}
+		.formatted-header {
+			margin-top: 40px;
+		}
+		.formatted-header__title {
+			font-size: 1rem;
+			color: #3c434a;
```

Comment:
We could probably use a variable for the color: I believe it'd be `--studio-color-gray-70`. Here's[ the page](https://color-studio.blog/) I use for reference.